intro-Linux/ch_uploading_files.tex

```diff
@@ -2,25 +2,29 @@ \chapter{Uploading/downloading/editing files}
 
 \subsection{Uploading/downloading files}
 
-Detailed information about uploading/downloading files with ``scp'' (using an OS
-X or Linux client) or WinSCP (Windows client), see
-\url{http://hpc.ugent.be/userwiki/index.php/User:VscCopy}.
+To transfer files from and to the HPC, see
+\href{\HPCManualURL#sec:filetranfer}{the section about tranferring files in the HPC manual}.
```

Comment:
Maybe mention `in Chapter 3 of the HPC manual`, so people can hunt for it if the anchor doesn't work in their PDF client?